docs/src/guide/processes.rst

```diff
@@ -53,6 +53,8 @@ ID of the child process.
 The exit callback will be invoked with the *exit status* and the type of *signal*
 which caused the exit.
 
+Note that it is important not to call ``uv_close`` before the exit callback.
```

Comment:
```suggestion
Note that it is important **not** to call ``uv_close`` before the exit callback.
```
Just a suggestion but otherwise it's easy to overlook that not.